libraries/Microsoft.Bot.Builder.Expressions/BuiltInFunctions.cs

```diff
@@ -1265,6 +1268,210 @@ private static string AddOrdinal(int num)
             return ordinalResult;
         }
 
+        // conversion functions
+        private static string ToBinary(string strToConvert)
+        {
+            var result = string.Empty;
+            foreach (var element in strToConvert.ToCharArray())
+            {
+                result += Convert.ToString(element, 2).PadLeft(8, '0');
+            }
+
+            return result;
+        }
+
+        private static (object, string) ToXml(object contentToConvert)
+        {
+            string error = null;
+            XDocument xml = null;
+            try
+            {
+                if (contentToConvert is string str)
+                {
+                    xml = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(Encoding.ASCII.GetBytes(str), new XmlDictionaryReaderQuotas()));
+                }
+                else
+                {
+                    xml = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(Encoding.ASCII.GetBytes(contentToConvert.ToString()), new XmlDictionaryReaderQuotas()));
+                }
```

Comment:
`var result = xml.ToString().TrimStart('{').TrimEnd('}');` should put here. If somethig is wrong , `xml` may be null, and `xml.ToString()` may throw exception